### ISSUE TITLE
Questions about clinical testing

### ISSUE THREAD

**Issue by hmyao22** [DenglinGo/bacteria-SANet#1]
Thank you for open sourcing your work. I used the script you provided to conduct reference pre-training for clinical testing, and then directly tested it on clinical 2018 without using fine-tuning. However, the accuracy is very different from the results reported in the article.
![image](https://github.com/DenglinGo/bacteria-SANet/assets/136300324/2f8876d5-56dd-491e-bd7f-f13dcd727c66)

Can you provide some solutions?

**Comment by hmyao22** [comment]
The provided pre-trained weight is also not work:
![image](https://github.com/DenglinGo/bacteria-SANet/assets/136300324/eeb17115-391d-4d46-b126-f6d63c6a9b42)
